samples/twitch-chat/extension/index.ts

```diff
@@ -13,7 +13,7 @@ module.exports = function (nodecg: NodeCG) {
     // Note that this does need a # before the channel name and is case-insensitive.
     const twitchChannels = ["#skate702", "#daniel0611"];
 
-    twitch?.requireService("sample", (client) => {
+    twitch?.requireService("Twitch-chat", (client) => {
```

Comment:
Same as with discord.
```suggestion
    twitch?.requireService("twitch-chat", (client) => {
```